package-res/resources/web/pentaho/util/BitSet.js

```diff
@@ -0,0 +1,92 @@
+define([
+  "pentaho/lang/Base"
+], function(Base) {
+
+  "use strict";
+
+  // Allow ~0
+  // jshint -W016
+
+  return Base.extend(/** @lends pentaho.util.BitSet# */{
+
+    /**
+     * The `BitSet` class represents a set data structure that is
+     * very efficient and can hold up to 31 elements, or bits.
+     *
+     * @alias BitSet
+     * @memberOf pentaho.util
+     * @class
+     * @extends pentaho.lang.Base
+     *
+     * @private
+     *
+     * @description Creates a bit set instance.
+     * @constructor
+     * @param {?number} [mask] - The bits to be set, initially. Defaults to no bits.
+     */
+    constructor: function(bits) {
+      this.set(bits || 0);
+    },
+
+    /**
+     * Returns a value that indicates if there are no bits set.
+     *
+     * @return {boolean} `true` if no bits are set, `false` otherwise.
+     */
+    get isEmpty() {
```

Comment:
For getters and setters we are using a different doc pattern like this:

/**
  * Gets the value that indicates if there are no bits set.
  *
  * @type boolean
  * @readonly
  */